test-infra/image-builder/roles/kubevirt-images/defaults/main.yml

```diff
@@ -69,6 +69,13 @@ images:
     converted: true
     tag: "latest"
 
+  rockeylinux-8:
```

Comment:
This part needs @floryut to run some manual action to build the image for rocky and push it to quay.io ... something to be added to an automated job